front/src/common/Pathfinding/Pathfinding.tsx

```diff
@@ -14,24 +14,14 @@ import { conditionalStringConcat, formatKmValue } from 'utils/strings';
 import { Path, PathQuery, osrdEditoastApi } from 'common/api/osrdEditoastApi';
 import { PointOnMap } from 'applications/operationalStudies/consts';
 
-import {
-  replaceVias,
-  updateItinerary,
-  updatePathfindingID,
-  updateSuggeredVias,
-} from 'reducers/osrdconf';
-import {
-  getInfraID,
-  getOrigin,
-  getDestination,
-  getVias,
-  getRollingStockID,
-  getPathfindingID,
-  getGeojson,
-} from 'reducers/osrdconf/selectors';
 import infraLogo from 'assets/pictures/components/tracks.svg';
 import InfraLoadingState from 'applications/operationalStudies/components/Scenario/InfraLoadingState';
-import { Spinner } from '../Loader';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { Spinner } from 'common/Loader';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
+import { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
import type { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Reply:
Done